electron/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -42,6 +42,12 @@ Categories=Game;
         writeFile(desktopFile, shortcut, () => {
           logInfo(`Shortcut saved on ${desktopFile}`, LogPrefix.Backend)
         })
+        chmod(desktopFile, '+x', () => {
```

Comment:
I think it's better call this insdie the `writeFile` callback, because if the write takes longer than expected the chmod command will try to set permissions to a yet non existing file

Reply:
fixed!